platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -21,112 +21,96 @@ describing the following:
 
 ## What is a platform
 
-A platform in distributed computing is a layer that provides common supporting
-capabilities and services for many uses. A platform provides consistent user
-experiences for getting, using and managing these capabilities and services,
-including Web portals and pages, scenario-specific code templates, automatable
-APIs and command-line tools.
+A platform as implemented for cloud-native computing is a layer that provides
+common supporting capabilities and services for many applications and use cases.
+Such a platform provides consistent user experiences for getting, using and
+managing its capabilities and services, including Web portals and pages,
+scenario-specific code templates, automatable APIs and command-line tools.
```

Comment:
I think it would be great to have our first sentence here not be cloud native focused. This is an opportunity for us to define a platform, and then a CNCF version of that separately.

Arguably, the CNCF bits could even go below the definitions provided by Atlassian/Fowler/Bottcher as those are still more generic. For example, on line 49 we are once again mentioning cloud native platforms and maybe the back half of this paragraph would work well in that location?

```suggestion
A platform is a layer that provides
common supporting capabilities and services for many applications and use cases.
A platform that supports cloud-native application development and delivery can provide 
consistent user experiences for getting, using and managing its capabilities and 
services, including Web portals and pages, scenario-specific code templates, 
automatable APIs and command-line tools.
```